app/lib/preload.js

```diff
@@ -105,7 +105,7 @@ function createBackend(ipcRenderer, platform) {
      * @param {string} event
      * @param {...any} args
      *
-     * @return {Promise<any>}
+     * @return {Promise<any>|string}
```

Comment:
Questionable API design, but indeed `string` is currently returned via `handleInPreload` :slightly_smiling_face: